### DISCUSSION TITLE
Inline editing

### DISCUSSION THREAD

**Discussion by megawubs** [TaKO8Ki/gobang#149] [Ideas]
I would like it to be able to edit values inline, and save them back to the database when I'm done editing.

Rough idea of the feature:

- Select a cell in the table
- Hit a key (my first ideas are `i` or `e`, `i` as it's the same in vim or `e` short for `edit`).
- When done, hit `esc` and the edited cell becomes marked with a colour
- Hit `w` (or another key-combination) to persist the changes

For editing the fields, vim key bindings would me nice. Thinking of this, is't it possible to use neovim remote (`:help remote.txt`) as the editing backend?

**Comment by sebashwa** [comment]
I'd love to see the same feature and I'd be happy to try and submit a PR, if desired.

I didn't come up with your suggestion to mark the cells and then submit the update afterwards though.
I can imagine this will make the implementation much harder compared with just submitting a cell update right away.
Can you elaborate on why you think it's favorable to mark multiple cells for edit and then submit afterwards?

**Comment by jorgerojas26** [comment]
i'd love to see this feature.
